### PR TITLE
[WIP] faq page

### PR DESCRIPTION
**Pull request by redonkulus**
@jedireza @mridgway

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+This is caused by the csrf middleware. You need to make sure you pass the `csrf` token to the `createContext` method on the server. Check the server.js of [chat](https://github.com/yahoo/flux-examples/blob/master/chat/server.js#L37) or [todo](https://github.com/yahoo/flux-examples/blob/master/todo/server.js#L41) examples.
+
+
+
+
+## Routing
+
+### Which should I use, `react-router` or `flux-router-component`?
+
+[react-router](https://github.com/rackt/react-router) has some features that flux-router-component does not have, like nested routes, component `willTransitionTo` hooks, and built-in re-directs. If you feel like you do not need them, then you will be fine with flux-router-component. We prefer the flux-like flow as you mentioned, so we use flux-router-component internally.
```
Comment (lingyan) [inline docs/faq.md]: "as you mentioned" where?

Comment (redonkulus) [inline docs/faq.md]: removed

**Comment by mridgway** [comment]
:+1:

**Pull request merged**